apps/studio/src/server/modules/site/site.router.ts

```diff
@@ -157,16 +170,46 @@ export const siteRouter = router({
       })
 
       await db.transaction().execute(async (tx) => {
-        await tx
+        const user = await tx
+          .selectFrom("User")
+          .where("id", "=", ctx.user.id)
+          .selectAll()
+          .executeTakeFirstOrThrow()
+
+        // Update site-level configuration
+        const oldSite = await tx
+          .selectFrom("Site")
+          .where("id", "=", siteId)
+          .selectAll()
+          .executeTakeFirstOrThrow()
```

Comment:
same here

Reply:
Updated in 489502187e9c9f571dc75a6b344c5ea98ce31990